tensorflow/python/kernel_tests/scatter_nd_ops_test.py

```diff
@@ -96,6 +96,14 @@ def _NumpyDiv(ref, indices, updates):
   return _NumpyScatterNd(ref, indices, updates, lambda p, u: p / u)
 
 
+def _NumpyMin(ref, indices, updates):
+  return _NumpyScatterNd(ref, indices, updates, min)
+
+
+def _NumpyMax(ref, indices, updates):
+  return _NumpyScatterNd(ref, indices, updates, max)
```

Comment:
Inputs are numpy arrays so `max` is not proper here. Should be `np.maximum`. Handled manually